docs/configuration.html

```diff
@@ -302,6 +302,22 @@ <h4><a id="org.apache.kafka.disallowed.login.modules"></a><a id="systempropertie
       <tr><th>Default Value:</th><td>com.sun.security.auth.module.JndiLoginModule</td></tr>
       </tbody></table>
     </li>
+    <li>
+      <h4><a id="org.apache.kafka.automatic.config.providers"></a><a id="systemproperties_org.apache.kafka.automatic.config.providers" href="#systemproperties_org.apache.kafka.automatic.config.providers">org.apache.kafka.automatic.config.providers</a></h4>
+      <p>This system property controls the automatic loading of ConfigProvider implementations in Apache Kafka. ConfigProviders are used to dynamically supply configuration values from sources such as files, directories, or environment variables. This property accepts a comma-separated list of ConfigProvider names. By default, all built-in ConfigProviders are enabled, including <b>FileConfigProvider</b>, <b>DirectoryConfigProvider</b>, and <b>EnvVarConfigProvider</b>.</p>
+      <p>If users want to disable all automatic ConfigProviders, they need to explicitly set the system property as shown below. Disabling automatic ConfigProviders is recommended in environments where configuration data comes from untrusted sources or where increased security is required. For more details, see <a href="https://nvd.nist.gov/vuln/detail/CVE-2024-31141">CVE-2024-31141</a>.</p>
```

Comment:
Should we link to https://kafka.apache.org/cve-list#CVE-2024-31141 instead?

Reply:
It is ok for me.  
Currently, I use the link from `https://nvd.nist.gov/`, to align with the  
[org.apache.kafka.disallowed.login.modules] (https://kafka.apache.org/documentation/#systemproperties_org.apache.kafka.disallowed.login.modules)。

If we decide to use the CVE link from the kafka-site, I suggest that we also update the link for `CVE-2023-25194.`

Reply:
I think it makes sense to point to our page which has a bit more details than the nist website. And our page has a link to nist.

Reply:
Make sense.
I have updated the link, PTAL 😺